### PR TITLE
Fix failing tests so this can be added to homebrew

### PR DESCRIPTION
**Pull request by clarkewd**
Fix failing tests so this can be added to homebrew - per errors / results at http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19933/testReport/

### CODE REVIEW

**Comment by jameskyle** [comment]
@clarkewd thanks!

File: openwsman.rb [hunk 1]
```diff
@@ -11,13 +9,19 @@ class Openwsman < Formula
...
-    system "./configure",
-                          "--disable-more-warnings",
+    system "./configure", "--disable-more-warnings",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    system "openwsmand --help"
```
Comment (mikemcquaid) [inline openwsman.rb]: This should be a non-`--help` test if that's at all possible.

File: openwsman.rb [hunk 2]
```diff
@@ -11,13 +9,19 @@ class Openwsman < Formula
   depends_on "autoconf"   => :build
   depends_on "libtool"    => :build
   depends_on "pkg-config" => :build
+  depends_on :openssl
```
Comment (mikemcquaid) [inline openwsman.rb]: Should be `"openssl"` not a symbol.

File: sblim-sfcc.rb [hunk 3]
```diff
@@ -8,11 +6,18 @@ class SblimSfcc < Formula
...
+  depends_on :openssl
+
   def install
     system "./configure","--prefix=#{prefix}", "--disable-dependency-tracking"
     system "make", "install"
   end
+
+  def test
+    # no executables, test to see if the man page has been installed
+    system "man -W CMCIClient"
```
Comment (mikemcquaid) [inline sblim-sfcc.rb]: Should import and use a header like e.g. `tinyxml.rb`

Comment (clarkewd) [inline sblim-sfcc.rb]: thanks for pointing us in the right direction @mikemcquaid - I updated these two tests per you suggestions - https://github.com/jameskyle/homebrew-rejected/pull/6

**Pull request merged**